### PR TITLE
Feature/skale 779 full sync node

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: node_cli/core/host.py [hunk 1]
```diff
@@ -69,12 +69,19 @@ def get_flask_secret_key():
...
+    env_type: str,
+    sgx_server_url: str = None,
+    allocation: bool = False
+):
     logger.info(f'Preparing host started, disk_mountpoint: {disk_mountpoint}')
     make_dirs()
     save_env_params(env_filepath)
-    save_sgx_server_url(sgx_server_url)
+
+    if sgx_server_url:
```
Comment (badrogger) [inline node_cli/core/host.py]: Let's remove it completely

File: node_cli/core/resources.py [hunk 2]
```diff
@@ -123,9 +123,7 @@ def generate_resource_allocation_config(env_file, force=False) -> None:
 
 
 def update_resource_allocation(disk_device: str, env_type: str) -> None:
-    resource_allocation_config = compose_resource_allocation_config(
-        disk_device, env_type
-    )
+    resource_allocation_config = compose_resource_allocation_config(env_type)
```
Comment (badrogger) [inline node_cli/core/resources.py:125]: We can remove disk_device from parameters

**Review by badrogger — CHANGES_REQUESTED** [review]

File: node_cli/core/host.py (hunk 1 above)
Comment (dmytrotkk) [reply node_cli/core/host.py]: done

File: node_cli/core/resources.py (hunk 2 above)
Comment (dmytrotkk) [reply node_cli/core/resources.py:125]: it's already removed

**Review by badrogger — APPROVED** [review]

**Pull request merged**